### PR TITLE
Adding `gitignore` Section to "Advanced Topics"

### PR DESCRIPTION
**Pull request by njlyon0**
First pass at this and comments welcome!

### CODE REVIEW

**Comment by angelchen7** [comment]
Nice work 👍 ! I really like the material you've added! Here are some minor things I noticed:

- The "\<your pick\>" part is not showing up in the rendered website so you may have to escape `<` and `>` with `\`:
`Once you find the option you want, click it and the template dropdown should change from ".gitignore template: None" to ".gitignore template: \<your pick\>".`
- In the "Exceptions to the .gitignore" section, are we assuming that the user already added the file name to the `.gitignore` before doing the 3 steps? And now they're trying to get `git` to untrack it by deleting and then adding it back in? I'm a bit confused on what kind of situation the user is in 🤔  hope that makes sense.

**Comment by angelchen7** [comment]
Great job on clarifying the "Exceptions to the .gitignore" section!

File: git_ignore.qmd [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+<img src="images/git_ignore/ignore-3-template-search.png" width = "70%"/>
+</p>
+
+Unfortunately for R users, typing just "R" returns all templates that contain the letter 'r' so you will need to scroll a bit to find the option that is actually for the R Statistical Environment. Once you find the option you want, click it and the template dropdown should change from ".gitignore template: None" to ".gitignore template: \<your pick\>".
+
+<p align="center">
+<img src="images/git_ignore/ignore-4-template-pick.png" width = "70%"/>
+</p>
+
+If you create a repository without choosing a `.gitignore` template, don't worry! A less detailed one will be created automatically when you clone your repository to your local computer. You can then add whatever information you want and make it contain the same information as the starter template.
```
Comment (brunj7) [inline git_ignore.qmd]: RStudio does create / modify a .gitignore file when you create an RStudio project, but it is not true when the command line or other tools.

File: git_ignore.qmd [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+3. Preserve Team Members' Local Filing Decisions
+
+    - Most working group members are involved in a staggering variety of pursuits in addition to their role in a synthesis working group. People tend to have their own filing systems in their computers to help them quickly and easily navigate among their different projects
+    - If you **commit** the .Rproj, you force everyone who clones your repository to use your naming convention which can be a hurdle (albeit likely a minor one) as they juggle their responsibilities
+
+#### Decision
+
+Ultimately, this is up to you and your team to decide! Hopefully our outlined reasons for and against adding the .Rproj to the `.gitignore` help inform this decision and we are happy to discuss this more if you have follow up questions!
+
+## Exceptions to the `.gitignore`
```
Comment (brunj7) [inline git_ignore.qmd]: I am not sure about "exceptions" in this section title? Seems more like a use case / example to me.

**Review by brunj7 — COMMENTED** [review]
Hi @njlyon0 ,

Thank you for putting this together. I made some minor comments and two commit with one suggestion really. Let me know what you think!

File: git_ignore.qmd (hunk 2 above)
Comment (njlyon0) [reply git_ignore.qmd]: Totally fair! Ping me when you're done making edits and I'll tweak this subheading accordingly!

File: git_ignore.qmd (hunk 1 above)
Comment (njlyon0) [reply git_ignore.qmd]: Good caveat, I'll make this tweak once you're finished with edits

**Pull request merged**